samza-core/src/test/java/org/apache/samza/processor/TestStreamProcessor.java

```diff
@@ -61,6 +61,7 @@ public TestableStreamProcessor(
     @Override
     SamzaContainer createSamzaContainer(
```

Comment:
If this is the existing StreamProcessor interface method that are used by existing customers, this would be a breaking change and we should consider to start the 0.14 branch for this PR.